### DISCUSSION TITLE
I2S Speaker for QT Py

### DISCUSSION THREAD

**Discussion by technoblogy** [pschatzmann/arduino-audio-tools#810] [Show and tell]
This is an expansion board for QT Py microcontroller boards to provide a 3W stereo power amplifier output from the digital I2S output:

![speaker3](https://user-images.githubusercontent.com/22325291/234793094-b0f57938-dacf-4d55-876f-0b4b556e24fe.jpg)

It includes examples using Arduino Audio Tools:

[I2S Speaker for QT Py](http://www.technoblogy.com/show?4ECO)

**Comment by pschatzmann** [comment]
Cool, I like this board!

**Reply by technoblogy** [reply]
Thanks!

**Comment by john-** [comment]
Thanks for publishing this project.  It really helps me see how the pieces fit together!

**Reply by technoblogy** [reply]
Glad it was useful!
